src/main/java/emu/grasscutter/game/sword/Sword.java

```diff
@@ -0,0 +1,5 @@
+package emu.grasscutter.game.sword;
+
+public class Sword {
+    //TODO: Add sword to world.
```

Comment:
No idea what `Sword` and `SwordManager` here refer to but this is just placeholder boilerplate and definitely doesn't belong in this PR.